pages/implementation/reference/javascript.mdx

```diff
@@ -0,0 +1,190 @@
+---
+layout: 'guide'
+title: 'JavaScript'
+---
+
+import Link from '../../../components/Link';
+
+### Platforms
+
+Avo can generate JavaScript code targeted at the following platforms
+
+- Web
+- React Native
+- Node.js
+
+### Quickstart
+
+Avo functions usage consists of 3 steps.
+
+#### Step 1. Include the Avo file
+
+##### Avo CLI is available through npm
+
+To install it and get the generated Avo file run the following commands.
+
+```bash
+$ npm install -g avo
+$ avo login
+$ avo pull --branch my-branch-name
+```
+
+Learn more about <Link passHref href="/implementation/cli"><a>the CLI here</a></Link>.
+
+You can also <Link passHref href="/explore-tracking-plan/download-or-copy-avo-file-manually"><a>download the file manually</a></Link> from your Avo workspace.
+
+#### Step 2. Initialize Avo
+
+Import Avo from the generated file and initialize it by calling the `initAvo` method before tracking
+
+```javascript
+function initAvo(options, systemProperties, 
+                 destinationOptions, [customDestDestination])
```

Comment:
```suggestion
import Avo from './Avo.js'

Avo.initAvo({ env: 'dev' }, [systemProperties], 
                [destinationOptions], [customDestDestination])
```

Reply:
I had it like this at first, but then thought that the import is actual code and method is a signature 🤔 Does this look ok to you?

Reply:
I think JS developers like reading the example-ish syntax better than the `function`. The `function` feels more like the implementation of the method rather than the usage. We could skip the import and stick with `Avo.initAvo?